### ISSUE TITLE
`Bug`: Next run in "True" hours

### ISSUE THREAD

**Issue by robertdahlem** [gyptazy/ProxLB#126]
## General
Set schedule: to 1 and see this in the log
`ProxLB: Info: [daemon]: Running in daemon mode. Next run in True hours.`

## Weighting
Score: 0

## Config
```
[service]
schedule: 1
```

## Meta
Version: 1.0.5
Installed from: https://cdn.gyptazy.com/files/os/debian/proxlb/
Running as: local on Proxmox

**Comment by gyptazy** [comment]
Hey @robertdahlem,

thanks for the report, but I’m wondering because this was fixed with #117 - are you sure version 1.0.5 is executed on the host?

cheers,
gyptazy

**Comment by robertdahlem** [comment]
> thanks for the report, but I’m wondering because this was fixed with #117 - are you sure version 1.0.5 is executed on the host?

```
# dpkg -l proxlb
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version      Architecture Description
+++-==============-============-============-=================================================================
ii  proxlb         1.0.5        amd64        ProxLB - Rebalance VM workloads across nodes in Proxmox clusters.

```

**Comment by gyptazy** [comment]
Ah, sorry - it’s part of 1.0.6, so yeah 1.0.5 is affected.

**Comment by gyptazy** [comment]
Fixed with release 1.0.6
